apps/web-mzima-client/src/app/settings/surveys/survey-item/survey-item.component.ts

```diff
@@ -94,7 +100,6 @@ export class SurveyItemComponent extends BaseComponent implements OnInit {
 
   private initLanguages(enabledLanguages: SurveyItemEnabledLanguages) {
     this.languages = this.languageService.getLanguages();
-    this.defaultLanguage = this.languages.find((lang) => lang.code === enabledLanguages.default);
```

Comment:
This is still needed in case someone has set another language as default, not english.